src/Validator.php

```diff
@@ -126,7 +127,10 @@ public function validate($data)
                 if (is_string($value) && (!isset($ruleProperties['trim']) || $ruleProperties['trim'] === true)) {
                     $value = trim($value);
                 }
-            // Don't validate empty values that are not set and not required
+                // allow a nullable value to be added as validated data
+            } elseif (!empty($ruleProperties['nullable']) && array_key_exists($fieldName, $data)) {
+                $value = null;
```

Comment:
Will this not set _all_ $values with `nullable => true` to `null`? 🤔 
So for example - if I send in `foo-bar` to a field with validator rule `nullable => true` that fields value will always be set to `null`

Reply:
no, that case is covered in the first if clause